<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kryptonise</class>
 <widget class="qMRMLWidget" name="Kryptonise">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>766</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Inputs</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="inputLabel">
        <property name="toolTip">
         <string>Pick the input labelmap, from which the mesh will be extracted.</string>
        </property>
        <property name="text">
         <string>Input label map:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="inputSelector" native="true">
        <property name="toolTip">
         <string>Pick the input labelmap, from which the mesh will be extracted.</string>
        </property>
        <property name="nodeTypes" stdset="0">
         <stringlist notr="true">
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes" stdset="0">
         <bool>false</bool>
        </property>
        <property name="addEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="removeEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>inputLabelMap</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="outputsCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Outputs</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="outputLabel">
        <property name="toolTip">
         <string>Select the output mesh node.</string>
        </property>
        <property name="text">
         <string>Output mesh:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="outputSelector" native="true">
        <property name="toolTip">
         <string>Select the output mesh node.</string>
        </property>
        <property name="nodeTypes" stdset="0">
         <stringlist notr="true">
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes" stdset="0">
         <bool>false</bool>
        </property>
        <property name="noneEnabled" stdset="0">
         <bool>true</bool>
        </property>
        <property name="addEnabled" stdset="0">
         <bool>true</bool>
        </property>
        <property name="removeEnabled" stdset="0">
         <bool>true</bool>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>outputMesh</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="parametersCollapsibleButton" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>306</height>
      </size>
     </property>
     <property name="text" stdset="0">
      <string>Parameters</string>
     </property>
     <property name="checked" stdset="0">
      <bool>true</bool>
     </property>
     <property name="collapsed" stdset="0">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="targetVertexCountLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Select the desired number of vertices of the output mesh.</string>
        </property>
        <property name="text">
         <string>Target vertex count</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ctkSliderWidget" name="targetVertexCountSliderWidget" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Select the desired number of vertices of the output mesh.</string>
        </property>
        <property name="decimals" stdset="0">
         <number>0</number>
        </property>
        <property name="singleStep" stdset="0">
         <double>1.000000000000000</double>
        </property>
        <property name="pageStep" stdset="0">
         <double>10000.000000000000000</double>
        </property>
        <property name="minimum" stdset="0">
         <double>1000.000000000000000</double>
        </property>
        <property name="maximum" stdset="0">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value" stdset="0">
         <double>100000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QGroupBox" name="RotoTranslationGroupBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>Roto-translation parameters</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="rollLabel">
           <property name="text">
            <string>Roll</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="ctkSliderWidget" name="rollSliderWidget" native="true">
           <property name="toolTip">
            <string>Set threshold value for computing the output image. Voxels that have intensities lower than this value will set to zero.</string>
           </property>
           <property name="decimals" stdset="0">
            <number>6</number>
           </property>
           <property name="singleStep" stdset="0">
            <double>1.000000000000000</double>
           </property>
           <property name="pageStep" stdset="0">
            <double>10.000000000000000</double>
           </property>
           <property name="minimum" stdset="0">
            <double>-180.000000000000000</double>
           </property>
           <property name="maximum" stdset="0">
            <double>180.000000000000000</double>
           </property>
           <property name="value" stdset="0">
            <double>-0.055268000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="pitchLabel">
           <property name="text">
            <string>Pitch</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="ctkSliderWidget" name="pitchSliderWidget" native="true">
           <property name="toolTip">
            <string>Set threshold value for computing the output image. Voxels that have intensities lower than this value will set to zero.</string>
           </property>
           <property name="decimals" stdset="0">
            <number>6</number>
           </property>
           <property name="singleStep" stdset="0">
            <double>1.000000000000000</double>
           </property>
           <property name="pageStep" stdset="0">
            <double>10.000000000000000</double>
           </property>
           <property name="minimum" stdset="0">
            <double>-90.000000000000000</double>
           </property>
           <property name="maximum" stdset="0">
            <double>90.000000000000000</double>
           </property>
           <property name="value" stdset="0">
            <double>0.026555000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="yawLabel">
           <property name="text">
            <string>Yaw</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="ctkSliderWidget" name="yawSliderWidget" native="true">
           <property name="toolTip">
            <string>Set threshold value for computing the output image. Voxels that have intensities lower than this value will set to zero.</string>
           </property>
           <property name="decimals" stdset="0">
            <number>6</number>
           </property>
           <property name="singleStep" stdset="0">
            <double>1.000000000000000</double>
           </property>
           <property name="pageStep" stdset="0">
            <double>10.000000000000000</double>
           </property>
           <property name="minimum" stdset="0">
            <double>-180.000000000000000</double>
           </property>
           <property name="maximum" stdset="0">
            <double>180.000000000000000</double>
           </property>
           <property name="value" stdset="0">
            <double>-83.063761000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="xLabel">
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="ctkSliderWidget" name="xSliderWidget" native="true">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string>Set threshold value for computing the output image. Voxels that have intensities lower than this value will set to zero.</string>
           </property>
           <property name="decimals" stdset="0">
            <number>5</number>
           </property>
           <property name="singleStep" stdset="0">
            <double>0.001000000000000</double>
           </property>
           <property name="pageStep" stdset="0">
            <double>1.000000000000000</double>
           </property>
           <property name="minimum" stdset="0">
            <double>-1000.000000000000000</double>
           </property>
           <property name="maximum" stdset="0">
            <double>1000.000000000000000</double>
           </property>
           <property name="value" stdset="0">
            <double>-0.080496000000000</double>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="yLabel">
           <property name="text">
            <string>y</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="ctkSliderWidget" name="ySliderWidget" native="true">
           <property name="toolTip">
            <string>Set threshold value for computing the output image. Voxels that have intensities lower than this value will set to zero.</string>
           </property>
           <property name="decimals" stdset="0">
            <number>5</number>
           </property>
           <property name="singleStep" stdset="0">
            <double>0.001000000000000</double>
           </property>
           <property name="pageStep" stdset="0">
            <double>1.000000000000000</double>
           </property>
           <property name="minimum" stdset="0">
            <double>-1000.000000000000000</double>
           </property>
           <property name="maximum" stdset="0">
            <double>1000.000000000000000</double>
           </property>
           <property name="value" stdset="0">
            <double>0.000005000000000</double>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="zLabel">
           <property name="text">
            <string>z</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="ctkSliderWidget" name="zSliderWidget" native="true">
           <property name="toolTip">
            <string>Set threshold value for computing the output image. Voxels that have intensities lower than this value will set to zero.</string>
           </property>
           <property name="decimals" stdset="0">
            <number>5</number>
           </property>
           <property name="singleStep" stdset="0">
            <double>0.001000000000000</double>
           </property>
           <property name="pageStep" stdset="0">
            <double>1.000000000000000</double>
           </property>
           <property name="minimum" stdset="0">
            <double>-1000.000000000000000</double>
           </property>
           <property name="maximum" stdset="0">
            <double>1000.000000000000000</double>
           </property>
           <property name="value" stdset="0">
            <double>30.239567999999998</double>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="scannerComboBox">
           <item>
            <property name="text">
             <string>default</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="scannerLabel">
           <property name="text">
            <string>Scanner</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="QPushButton" name="savePushButton">
           <property name="text">
            <string>Save as new default</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="debugLevelLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Select the desired number of vertices of the output mesh.</string>
        </property>
        <property name="text">
         <string>Debug Level</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="ctkSliderWidget" name="debugLevelSliderWidget" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Select the desired number of vertices of the output mesh.</string>
        </property>
        <property name="decimals" stdset="0">
         <number>0</number>
        </property>
        <property name="singleStep" stdset="0">
         <double>1.000000000000000</double>
        </property>
        <property name="pageStep" stdset="0">
         <double>1.000000000000000</double>
        </property>
        <property name="minimum" stdset="0">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum" stdset="0">
         <double>3.000000000000000</double>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="applyButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>outputsCollapsibleButton</zorder>
  <zorder>parametersCollapsibleButton</zorder>
  <zorder>inputsCollapsibleButton</zorder>
  <zorder>applyButton</zorder>
  <zorder>verticalSpacer</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>Kryptonise</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Kryptonise</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>outputSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
